source/iface/range.rst

```diff
@@ -28,14 +28,28 @@ It can be constructed from integers.
 
   range();
 
+Construct a ``SYCL`` range with the value ``0`` for each dimension
+
+::
+
   range(size_t dim0);
 
+Construct a 1D ``SYCL`` with value ``dim0``.
```

Comment:
The word "range" is missing here.  I think you either meant to say:

> Construct a 1D ``SYCL`` range with value ``dim0``.

or maybe:

> Construct a 1D range with value ``dim0``.

(I'm not sure we need to keep repeating the word "SYCL" for all of these function descriptions.)

Same comment for the next two function descriptions.

Reply:
yes, copy/paste going wrong ... Fixed this.

BTW, most of the wording in this patch comes from the spec, so if it looks ok to you, I'll mirror the changes there as well.